src/content/admin/cleanup.mdx

```diff
@@ -117,3 +220,104 @@ In that case, the Garbage Collector will ignore only those specific resources wh
+
+## Troubleshooting
+
+### Common Issues and Solutions
+
+**Problem: Garbage Collector Not Sleeping Resources**
+
+*Symptoms:* Resources remain active despite exceeding the sleep period
+
+*Solutions:*
+1. **Check GC Status**: Go to **Admin → Garbage Collector** and verify it's enabled
+2. **Verify Timing**: Confirm the sleep period configuration and when the resource was last active
+3. **Check for Activity**: Look for recent deployments, `okteto up` sessions, or kubectl operations
+4. **Review Labels**: Ensure resources don't have `dev.okteto.com/persistent` label
+5. **Check Logs**: Look in **Admin → System Logs** for GC errors or warnings
+
+**Problem: Resources Deleted Unexpectedly**
+
+*Symptoms:* Important resources were deleted by GC
+
+*Solutions:*
+1. **Mark as Persistent**: Add `dev.okteto.com/persistent` label to critical namespaces
+2. **Adjust Periods**: Increase delete periods for your use case
+3. **Check Activity Tracking**: Ensure team knows what counts as "activity"
+4. **Recovery**: If recently deleted, check if backups are available
+
+**Problem: Personal Namespaces Being Affected**
+
+*Symptoms:* Personal Namespace contents are being deleted
+
+*Solutions:*
+1. **Check Settings**: Verify **Personal Namespaces GC** toggle in Admin → Garbage Collector
+2. **Understand Scope**: Remember that Personal Namespaces contents are cleaned, not the namespaces themselves
+3. **PVC Protection**: Confirm PVCs are preserved as expected
+
+**Problem: Auto-wake Not Working**
+
+*Symptoms:* Incoming requests don't wake sleeping namespaces
+
+*Solutions:*
+1. **Check Ingress**: Ensure you're using Okteto's Nginx Ingress Controller
+2. **Verify Configuration**: Check if `autowake` is disabled in Helm config
+3. **DNS/Routing**: Confirm requests are reaching the correct ingress endpoint
+
+
+## Best Practices and Use Cases
+
+### Recommended Settings by Team Size
+
+**Small Teams (5-15 developers):**
+- **Namespaces**: Sleep 6h, Delete 21d  
+- **Preview Environments**: Sleep 2h, Delete 3d
+- **Reasoning**: Longer periods account for time zones and varying work schedules
+
+**Medium Teams (15-50 developers):**
+- **Namespaces**: Sleep 4h, Delete 14d
+- **Preview Environments**: Sleep 1h, Delete 2d  
+- **Reasoning**: More active development requires faster cleanup of unused resources
+
+**Large Teams (50+ developers):**
+- **Namespaces**: Sleep 3h, Delete 10d
+- **Preview Environments**: Sleep 30min, Delete 1d
+- **Reasoning**: High resource utilization demands aggressive cleanup
+
+
+### Cost Optimization Strategies
+
+**1. Implement Graduated Cleanup:**
+- Short sleep periods (1-2h) for immediate cost savings
+- Moderate delete periods (7-14d) for safety
+- Monitor and adjust based on usage patterns
+
+**2. Use Labels Strategically:**
+```yaml
+# Mark production-like environments as persistent
+kubectl label namespace staging dev.okteto.com/persistent=true
```

Comment:
I would encourage customers to use Okteto's UI for this. Individual resources cannot be marked as persistent from the UI, but namespaces can (only admins).

So, I would say that admins can mark an entire namespace as persistent from the UI, but if they need more granularity and make persistent only specific/critical resources, they can do that adding the label at k8s level to that resource